packages/react-server-dom-webpack/src/ReactFlightWebpackPlugin.js

```diff
@@ -172,25 +172,36 @@ export default class ReactFlightWebpackPlugin {
       const json = {};
       compilation.chunkGroups.forEach(chunkGroup => {
         const chunkIds = chunkGroup.chunks.map(c => c.id);
+
+        function recordModule(id, mod) {
+          // TODO: Hook into deps instead of the target module.
+          // That way we know by the type of dep whether to include.
+          // It also resolves conflicts when the same module is in multiple chunks.
+          if (!/\.client\.js$/.test(mod.resource)) {
+            return;
+          }
+          const moduleExports = {};
```

Comment:
Is reduce a cleaner way to create moduleExports?
```
const names = ['', '*'].concat(mod.buildMeta.providedExports);
const moduleExports = names.reduce((accumulator, name) => {
  accumulator[name] = {
    id: id,
    chunks: chunkIds,
    name: name,
  };
  
  return accumulator;
}, {});
```

Reply:
Thanks for the suggestion, but I don't think this makes a difference here.